mteb/model_meta.py

```diff
@@ -1,24 +1,26 @@
 from __future__ import annotations
 
-from datetime import date
+import logging
 from functools import partial
-from typing import TYPE_CHECKING, Annotated, Any, Callable, Literal
+from typing import TYPE_CHECKING, Any, Callable, Literal
 
-from pydantic import BaseModel, BeforeValidator, TypeAdapter
+from pydantic import BaseModel
 
+from mteb.abstasks.TaskMetadata import STR_DATE, STR_URL
 from mteb.encoder_interface import Encoder
 
 from .languages import ISO_LANGUAGE_SCRIPT
 
 if TYPE_CHECKING:
     from .models.sentence_transformer_wrapper import SentenceTransformerWrapper
 
-Frameworks = Literal["Sentence Transformers", "PyTorch"]
+logger = logging.getLogger(__name__)
 
-pastdate_adapter = TypeAdapter(date)
-STR_DATE = Annotated[
-    str, BeforeValidator(lambda value: str(pastdate_adapter.validate_python(value)))
-]  # Allows the type to be a string, but ensures that the string is a valid date
+
+FRAMEWORKS = Literal[
+    "Sentence Transformers", "PyTorch", "GritLM", "LLM2Vec", "TensorFlow", "API"
```

Comment:
I think [`Tevatron`](https://github.com/texttron/tevatron/tree/main) is another one that many people have used

Reply:
Maybe also AnglE?